src/dune_rules/expander.ml

```diff
@@ -426,28 +422,6 @@ let lib_config_var (var : Pform.Var.t) (lib_config : Lib_config.t) =
   ]
 ;;
 
-let ocaml_config_var (var : Pform.Var.t) (ocaml_config : Ocaml_config.t) =
```

Comment:
Where did this code go? Was it duplicated?

Reply:
ah oops I forgot to add a file. It's in https://github.com/ocaml/dune/pull/9750/files#diff-5e4f430877c0a281dea7b3d55853b8a5743863c1528cc4b64c9fccebb25bc9b6